apps/website/src/components/table-issues/table-issues.tsx

```diff
@@ -0,0 +1,306 @@
+  },
+  {
+    id: 873,
+    title: 'Add support for encrypted communities',
+    status: 'open',
+  },
+  {
+    id: 123,
+    title: 'Add support for encrypted communities',
+    status: 'open',
+  },
+]
+
+const authors: FilterWithCheckboxesProps['data'] = [
+  {
+    id: 4,
+    name: 'marcelines',
+    avatar: 'https://avatars.githubusercontent.com/u/29401404?v=4',
+  },
+  {
+    id: 5,
+    name: 'prichodko',
+    avatar: 'https://avatars.githubusercontent.com/u/14926950?v=4',
+  },
+  {
+    id: 6,
+    name: 'felicio',
+    avatar: 'https://avatars.githubusercontent.com/u/13265126?v=4',
+  },
+  {
+    id: 7,
+    name: 'jkbktl',
+    avatar: 'https://avatars.githubusercontent.com/u/520927?v=4',
+  },
+]
+
+const epics: FilterWithCheckboxesProps['data'] = [
+  {
+    id: 4,
+    name: 'E:ActivityCenter',
+    color: '$orange-60',
+  },
+  {
+    id: 5,
+    name: 'E:Keycard',
+    color: '$purple-60',
+  },
+  {
+    id: 6,
+    name: 'E:Wallet',
+    color: '$pink-60',
+  },
+  {
+    id: 7,
+    name: 'E:Chat',
+    color: '$beige-60',
+  },
+]
+
+const labels: FilterWithCheckboxesProps['data'] = [
+  {
+    id: 4,
+    name: 'Mobile',
+    color: '$blue-60',
+  },
+  {
+    id: 5,
+    name: 'Frontend',
+    color: '$brown-50',
+  },
+  {
+    id: 6,
+    name: 'Backend',
+    color: '$red-60',
+  },
+  {
+    id: 7,
+    name: 'Desktop',
+    color: '$green-60',
+  },
+]
+
+const assignees: FilterWithCheckboxesProps['data'] = [
+  {
+    id: 1,
+    name: 'Unassigned',
+    avatar: <ProfileIcon size={16} />,
+  },
+  {
+    id: 4,
+    name: 'marcelines',
+    avatar: 'https://avatars.githubusercontent.com/u/29401404?v=4',
+  },
+  {
+    id: 5,
+    name: 'prichodko',
+    avatar: 'https://avatars.githubusercontent.com/u/14926950?v=4',
+  },
+  {
+    id: 6,
+    name: 'felicio',
+    avatar: 'https://avatars.githubusercontent.com/u/13265126?v=4',
+  },
+  {
+    id: 7,
+    name: 'jkbktl',
+    avatar: 'https://avatars.githubusercontent.com/u/520927?v=4',
+  },
+]
+
+const repositories: FilterWithCheckboxesProps['data'] = [
+  {
+    id: 1,
+    name: 'status-mobile',
+  },
+  {
+    id: 2,
+    name: 'status-desktop',
+  },
+  {
+    id: 3,
+    name: 'status-web',
+  },
+  {
+    id: 4,
+    name: 'status-go',
+  },
+  {
+    id: 5,
+    name: 'nwaku',
+  },
+  {
+    id: 6,
+    name: 'go-waku',
+  },
+  {
+    id: 7,
+    name: 'js-waku',
+  },
+  {
+    id: 8,
+    name: 'nimbus-eth2',
+  },
+  {
+    id: 9,
+    name: 'help.status.im',
+  },
+]
+
+const sortOptions: SortProps['data'] = [
+  {
+    id: 1,
+    name: 'Default',
+  },
+  {
+    id: 2,
+    name: 'Alphabetical',
+  },
+  {
+    id: 3,
+    name: 'Creation date',
+  },
+  {
+    id: 4,
+    name: 'Updated',
+  },
+  {
+    id: 5,
+    name: 'Completion',
+  },
+]
+
+const TableIssues = () => {
+  const [issuesSearchText, setIssuesSearchText] = useState('')
+  const inputRef = useRef<TextInput>(null)
+  const [isMinimized, setIsMinimized] = useState(true)
```

Comment:
This can be moved to the Input component if no-one relies on this outside of it.